src/js/modules/SelectRange/SelectRange.js

```diff
@@ -758,7 +759,7 @@ export default class SelectRange extends Module {
 	redraw(force) {
 		if (force) {
 			this.selecting = 'cell';
-			this.resetRanges();
+			this.resetRanges(false);
```

Comment:
If you can make the new `setInitialRange` method, you don't need to pass this boolean flag.

Reply:
yes, the flag is gone no. (see also other comments)